setup.cfg

```
@@ -40,7 +40,7 @@ install_requires =
     statsmodels >= 0.10
     graphviz
     pandas
-    shap ~= 0.38.1
+    shap >= 0.38.1, <= 0.39.0
```

I think we'd expect point releases of shap to remain okay if the 0.39.0 release is okay, so a more liberal dependency of

```suggestion
    shap >= 0.38.1, < 0.40
```

would be okay, too.

---

You got it!